app/helpers/dwp_outcome_helper.rb

```diff
@@ -0,0 +1,11 @@
+module DWPOutcomeHelper
+  def reset_confirm_dwp_status(legal_aid_application)
```

Comment:
I think these methods should have bangs as they call save!

```suggestion
  def reset_confirm_dwp_status!(legal_aid_application)
```